src/porepy/numerics/ad/discretizations.py

```diff
@@ -376,7 +376,8 @@ def _flux_function(
         # The first part is rather involved and is handled inside self._transmissibility
 
         # Get hold of the underlying flux discretization.
-        base_flux = self._discretization.flux.evaluate(self.equation_system)
+        base_flux = self._discretization.flux.value(self.equation_system)
+        assert isinstance(base_flux, np.ndarray)
```

Comment:
I'm pretty sure this is a sparse matrix (the transmissibility matrix.)

Reply:
Fixed, thanks